text/0000-serialization-format.md

```diff
@@ -0,0 +1,95 @@
+# Serialization Format
+
+## Summary
+
+[summary]: #summary
+
+Definition of a versioned, backwards-compatible and extensible serialization format for cartographer-maps.
+
+This RFC is a small deviation of the previously proposed header format from [RFC-0015](https://gitub.com/googlecartographer/rfc/text/0015-serialization-header.md).
```

Comment:
the link is broken because of the typo: "gitub"

Reply:
fixed.